osgp/protocol-adapter-mqtt/osgp-protocol-adapter-mqtt/src/main/java/org/opensmartgridplatform/adapter/protocol/mqtt/domain/entities/MqttDevice.java

```diff
@@ -62,6 +62,10 @@ public String getTopics() {
     return this.topics;
   }
 
+  public String[] getTopicsArray() {
```

Comment:
`getTopics` (above this function) is only used (incorrectly...) by Cucumber step, might be a good idea to clean it up.

Reply:
Refactored `getTopics` / `setTopics` to use `String[]`.